commcare/src/test/java/org/motechproject/commcare/service/impl/CommcareCaseServiceImplTest.java

```diff
@@ -95,6 +95,22 @@ public void testCasesByUserId() {
     }
 
     @Test
+    public void testCasesByOwnerId() {
+        String ownerId = "testId";
+
+        CaseRequest request = new CaseRequest();
+        request.setOwnerId(ownerId);
+        request.setLimit(20);
+        request.setOffset(0);
+        when(commcareHttpClient.casesRequest(config.getAccountConfig(), request)).thenReturn(casesResponse());
+
+        List<CaseInfo> cases = caseService.getCasesByOwnerId(ownerId, 20, 1);
```

Comment:
what I meant is to check the ownerId field in the testAllCases() test. So in other words, add a new assert to the existing test. We do not really need these new methods from what I understand.